temoa/extensions/stochastics/scenario_creator.py

```
@@ -0,0 +1,130 @@
+from temoa._internal.run_actions import build_instance
+from temoa.components.costs import period_cost_rule
+from temoa.data_io.hybrid_loader import HybridLoader
+
+if TYPE_CHECKING:
+    from temoa.core.config import TemoaConfig
+    from temoa.extensions.stochastics.stochastic_config import StochasticConfig
+
+logger = logging.getLogger(__name__)
+
+
+def scenario_creator(scenario_name: str, **kwargs: Any) -> pyo.ConcreteModel:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)